app/views/posts/show.html.erb

```diff
@@ -1,9 +1,19 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <title>Post Details</title>
-  </head>
-  <body>
-    <h1>Details for a given post</h1>
-  </body>
-</html>
+<div class="flex-col">
+  <div id="post" class="border-solid border-4 border-black md:w-1/2 mx-auto w-full mb-5">
+    <div class="flex justify-between">
+      <div>
+        <h2 class="font-bold ml-3"><%= @post.title %><span class="font-normal"> by <%= @user.name %></span></h2>
+      </div>
+
+      <div class="mr-5">
+        <p>comments: <span class="font-bold"><%= @post.comments_counter%>, </span>Likes: <span class="font-bold"><%= @post.likes_counter %></span></p>
+      </div>
```

Comment:
- [ ] Please move this counter section to the right side to match the given design.
![image](https://github.com/rmiyoyo/blog-app/assets/91486791/7abfdb5c-ee87-44ab-9b30-a5b9d49053ec)